### ISSUE TITLE
Missing ARM64 debian package, and snapcast repository

### ISSUE THREAD

**Issue by jeanbruder** [badaix/snapcast#706]
Dear Snapcast Team,

Whaou, what a great job ! It is very easy now to have multi-room audio without have to use PulseAudio configured, many thanks for the great work You made.

Would it be possible to have ARM64 packages available, as only ARMF and AMD64 packages are for now available ?
In that way, could it be possible to have a repository of the precompiled binaries for each architecture and the related .DEB packages ?

Many thanks in advance,
Best regards,
Jean

**Comment by badaix** [comment]
There should be official debian packages available for several architectures (maybe in some older version). 
I'm only providing packages that can be built automatically (with GitHub actions) and that I can test (i.e. for architectures/OS combinations that I have running at home). When it should happen that I update my RPi4 to a 64 bit Raspbian (earliest when it's officially available), I will install a self-hosted GitHub action runner on it to automatically build arm64 packages.

**Comment by jeanbruder** [comment]
Thx @badaix for your answer.

I compiled successfully this afternoon the latest version of snapcast on my board (arm64 : Rock64) without any issues. I installed the latest version available in the official Debian repository for my architectue (v 0.15) and replaced the binary files with the one compiled. I got no error to start the snapcast server.

But, I have many errors while playing the music ... I will open a case in that way.

Regards,
Jean
